Question3.java

```diff
@@ -0,0 +1,125 @@
+import java.util.Scanner;
+
+//Exception Classes
+class InvalidEntryException  extends Exception
+{
+    public InvalidEntryException(String message) {super(message);}
+}
+class CardSuiteException  extends Exception
+{
+    public CardSuiteException(String message) {super(message);}
+}
+
+//card_select class
+class card_select
```

Comment:
Have 1 class named Card.
It will have data members

1. suite
2. number

Have a constructor which takes both these values and does the validation.

Make the card class implement Comparable interface and have the sorting logic there.

In your main method create a list of cards of same suite and see if the sorting works with Collection.sort method
Try and compare 2 cards of different suite and see what happens.

Dont need to take input from user. Make static objects

Reply:
This issue has been resolved.